src/aspire/image/image.py

```diff
@@ -334,6 +349,20 @@ def backproject(self, rot_matrices):
 
         return aspire.volume.Volume(vol)
 
+    def show(self, columns=5, figsize=(20, 10)):
```

Comment:
I'm not a huge fan of this pattern, since it mixes in the UI functionality with our lower-level image processing stuff. On the other hand, I can't think of an elegant solution here…

Reply:
I can make a standalone function somewhere else that consumes Images (or other ASPIRE types) and tries to do a basic plot, but then people need know it exists and bring it in.  It is kind of annoying to have to construct matplotlib just to peek at images. I have been using this last few days and it is super convenient.

Are there common image processing libraries that don't have helper function to look at images?

Reply:
> I can make a standalone function somewhere else that consumes Images (or other ASPIRE types) and tries to do a basic plot, but then people need know it exists and bring it in. It is kind of annoying to have to construct matplotlib just to peek at images.

Right. That would be the alternative, but as you point out, not very elegant.

> Are there common image processing libraries that don't have helper function to look at images?

I'm not sure what's done elsewhere. Again, it's not clear that there even is a good solution here.

Part of the problem is that we're mixing levels (`Image` is ostensibly low-level code, while displaying images in the UI is high-level). Another is that we're messing with global (matplotlib) state, but that has more to do with matplotlib's design choices than anything else. One potential solution to this latter is to supply the plot object to the function, which SciPy does in some of its stats functions (https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.probplot.html).

At any rate, I think this is fine as is. It's mostly a debugging thing and it's not clear how to do it in a cleaner way. I'm resolving this.

Reply:
> I can make a standalone function somewhere else that consumes Images (or other ASPIRE types) and tries to do a basic plot, but then people need know it exists and bring it in. It is kind of annoying to have to construct matplotlib just to peek at images.

Right. That would be the alternative, but as you point out, not very elegant.

> Are there common image processing libraries that don't have helper function to look at images?

I'm not sure what's done elsewhere. Again, it's not clear that there even is a good solution here.

Part of the problem is that we're mixing levels (`Image` is ostensibly low-level code, while displaying images in the UI is high-level). Another is that we're messing with global (matplotlib) state, but that has more to do with matplotlib's design choices than anything else. One potential solution to this latter is to supply the plot object to the function, which SciPy does in some of its stats functions (https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.probplot.html).

At any rate, I think this is fine as is. It's mostly a debugging thing and it's not clear how to do it in a cleaner way. I'm resolving this.